tasks/Qc.wdl

```diff
@@ -318,6 +318,31 @@ task BuildBamIndex {
   }
 }
 
+# Collect BAM/CRAM index stats
+# NOTE: Samtools idxstats is slow on CRAM inputs since the index does not contain the info needed to summarize per-chromosome alignments
+task BamIndexStats {
+  input {
+    File input_bam
+    File input_bam_index
+    Int preemptible_tries
+  }
+
+  Int disk_size = ceil(size(input_bam, "GiB")) + 20
+
+  command {
+    samtools idxstats ~{input_bam} > ~{input_bam}.idxstats
```

Comment:
This has the same problem as #60 addressed.  We don't want to use the full path to the input BAM as the output location; we should grab its `basename` instead.

Reply:
Addressed similar to #60